server/protocols/webadmin/webadmin-mailbox/src/main/java/org/apache/james/webadmin/service/ExportService.java

```diff
@@ -83,6 +84,14 @@ public ExportService(MailboxBackup mailboxBackup, BlobStore blobStore, BlobExpor
                 .export()));
     }
 
+    private void deleteBlob(BlobId blobId) {
+        blobStore.delete(blobStore.getDefaultBucketName(), blobId)
```

Comment:
I belive we could be using the low level blob store along with a randomly generated blobId to avoid deduplication here?

Cc @remk  ?

Reply:
Affter discussing with @chibenwa, we came to the conclusion that we should go with the BlobStore instead of the DumbBlobStore for the time being because of two reasons:

- will cause conflict with a WIP from the Moka Team. see #3136 
- May requires modifications not only to the LocalFileExportMechanism class but also to both BlobStore interfaces/implementations, which will complicate things further.

We are looking forward to feedback concerning this issue from the Moka Team. It surely will be very helpful.